docs/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I am curious about why this file is here.  Did you create a web project in Eclipse for the site?  Is that how this file was generated?